MacPass/MPAutotypeDaemon.m

```diff
@@ -385,6 +391,9 @@ - (void)_presentSelectionWindow:(NSArray *)candidates {
     self.matchSelectionWindow.contentViewController = vc;
     
   }
+  if(self.targetPID) {
```

Comment:
Why do you want to move the target app to the front and present the selection window afterwards?

Reply:
No document open - global autofill - notification - click - enter password - display the fill window on MacPass, click fill item ,  display the target window, then disappear, I think when the password input is completed, SelectionWindow will be better on top of targetPID Better，Put another way, when the SelectionWindow is first displayed, the targetPID should be below it. Thanks for review